config/environment.js

```diff
@@ -181,6 +181,18 @@ module.exports = function (environment) {
 				region: 'us-west-2',
 				architectEnumString: 'archEnums.LOCATIONS.prod_eu_west_2',
 			},
+			appsaps1purecloud: {
+				clientId: '96a7d55b-1ed5-4719-9094-08a2a69ca07c',
+				redirect: 'https://apps.aps1.pure.cloud/developer-tools/',
+				region: 'ap-south-1',
+				architectEnumString: 'archEnums.LOCATIONS.prod_ap_south_1',
+			},
+			appsuse2usgovpurecloud: {
```

Comment:
shouldn't this have entries for developeraps1purecloud and developeruse2usgovpurecloud also?

Reply:
Added. I want to say I had my reason not to add those two, but maybe I just forgot 😑